### PR TITLE
fix BitmapText  textWidth

### PR DESCRIPTION
**Pull request by knoxHuang**

### CODE REVIEW

File: src/pixi/text/BitmapText.js [hunk 1]
```diff
@@ -141,12 +142,13 @@ PIXI.BitmapText.prototype.updateText = function()
 
         chars.push({texture:charData.texture, line: line, charCode: charCode, position: new PIXI.Point(pos.x + charData.xOffset, pos.y + charData.yOffset)});
         pos.x += charData.xAdvance;
+        lastLineWidth = pos.x + (charData.texture.width + charData.xOffset) - charData.xAdvance;
```
Comment (englercj) [inline src/pixi/text/BitmapText.js]:
```js
pos.x += charData.xAdvance;
lastLineWidth = pos.x + (charData.texture.width + charData.xOffset) - charData.xAdvance;
```

Why not just do:

```js
lastLineWidth = pos.x + (charData.texture.width + charData.xOffset);
pos.x += charData.xAdvance;
```

**Comment by englercj** [comment]
What problem is this solving? There is no description on this PR.

**Comment by knoxHuang** [comment]
Hi, I am trying to fix the problem that the BitmapText.textWidth not equals to its "sprite fonts' width". This PR caculates the width of the last sprite by using xOffset + charData.texture.width, nor xAdvance.

File: src/pixi/text/BitmapText.js (hunk 1 above)
Comment (knoxHuang) [inline src/pixi/text/BitmapText.js]: You can also do this

**Pull request closed without merging**